src/App.vue

```diff
@@ -10,6 +10,10 @@ import TopHeader from "./components/TopHeader.vue";
 
 export default {
   name: "App",
+  metaInfo: {
+    title: "",
```

Comment:
some fallback may be useful - it'll be used when some page does not provide it's own `title`. In such a case, currently page title would be `" | Monte Cinema"`